WikipediaUnitTests/WMFArticleSummaryVisualTests.m

```diff
@@ -72,17 +78,15 @@ - (void)verifySummaryForFixtureData:(NSDictionary*)mobileViewJSON langCode:(NSSt
                                            dataStore:nil
                                                 dict:mobileViewJSON[@"mobileview"]];
 
-    WMFMinimalArticleContentCell* testCell = [WMFMinimalArticleContentCell new];
-    testCell.attributedString = self.article.summaryHTML;
-
-    // attributed text cell need a superview else it won't layout its subviews
-    UIView* testView = [UIView new];
-    [testView addSubview:testCell];
-    testView.frame = (CGRect){
-        .origin = CGPointZero,
-        .size   = [testCell.attributedTextContextView suggestedFrameSizeToFitEntireStringConstraintedToWidth:320]
-    };
-    testCell.frame = testView.bounds;
+    WMFMinimalArticleContentCell* testCell = [self.articleVC.tableView dequeueReusableCellWithIdentifier:[WMFMinimalArticleContentCell wmf_nibName]];
```

Comment:
ah, clever, didn't think about grabbing the VC's cell directly. this is a much more "accurate" test